recipe/build.sh

```diff
@@ -10,9 +10,4 @@ echo "udunits2_path = $PREFIX/lib/libudunits2${SHLIB_EXT}" >> $SITECFG
 
 rm -rf lib/iris/tests/results lib/iris/tests/*.npz
 
-python setup.py install --single-version-externally-managed --record record.txt
-
-
-# Without this line, the OS X build fails reproducibly with
-# https://github.com/conda-forge/iris-feedstock/pull/2#issuecomment-217468453.
-python -c "import iris.fileformats.netcdf; iris.fileformats.netcdf._pyke_kb_engine()"
```

Comment:
@pelson and @djkirkham it seems that we are OK without the patch and this line.
I am guessing that https://github.com/SciTools/iris/issues/2004 was fixed.

Reply:
Yep. Thanks for that. I've updated the issue.